src/tusb_option.h

```diff
@@ -284,6 +284,11 @@
   #define CFG_TUD_DWC2_DMA_ENABLE CFG_TUD_DWC2_DMA_ENABLE_DEFAULT
 #endif
 
+// Allocate double buffer for IN endpoint for better performance (only applies to non-periodic endpoints)
+#ifndef CFG_TUD_DWC2_IN_DOUBLE_BUFFER
+  #define CFG_TUD_DWC2_IN_DOUBLE_BUFFER 0
```

Comment:
I think we should not introduce this option. I always think the fine-tune fifo should be dictated by application per endpoint basic. E.g MSC + CDC, we would want a massive endpoint size for msc and smaller for cdc. I am think we should do that via a tud_configure() (which allo to pass an option id + pointer to struct) that fine tune the behavior of dcd or stack. similar to tuh_configure() https://github.com/hathach/tinyusb/blob/master/src/host/usbh.h#L145 . dwc2 option may include grxsize and txfifo size for each endpoint (0 for auto as current).